src/CONSTANTS.ts

```diff
@@ -69,7 +79,7 @@ export let EVENT_COUNTER_ID = getGlobalId(network);
 
 function getIdPrefix(network: string): string {
   if (
-    network == "goerli" /* Only include goerli while testing */ ||
+    (network == "goerli" && maticTestingDeployment) ||
     network == "matic" ||
     network == "mumbai"
   ) {
```

Comment:
This just makes sure the 'network' prefixes that are needed to avoid confusion between the two graphs are set correctly